K2Bridge.Tests.UnitTests/Visitors/RangeAggregationVisitorTests.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace UnitTests.K2Bridge.Visitors
+{
+    using System.Collections.Generic;
+    using global::K2Bridge.Models.Request.Aggregations;
+    using global::K2Bridge.Tests.UnitTests.Visitors;
+    using global::K2Bridge.Visitors;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class RangeAggregationVisitorTests
+    {
+        [TestCase(0, 800, 800, 2000, ExpectedResult = "wibble by ['_alias'] = case(dayOfWeek >= 0 and dayOfWeek < 800, '0-800', dayOfWeek >= 800 and dayOfWeek < 2000, '800-2000', 'default_bucket')")]
```

Comment:
nit: In the latet version alias are no longer prefixed with '_'

Reply:
👍🏻